### DISCUSSION TITLE
How to trigger workflow when all checks pass for pull request (combined status)

### DISCUSSION THREAD

**Discussion by axsuul** [community/community#25535] [Actions]
<p>I’m interested in creating a workflow to notify Slack whenever a pull request’s combined status is success. I tried looking through all the possible events. The one that gets close is <code>check_runs</code> but wouldn’t fly if I have multiple of those. Is this possible?</p>

**Comment by gimenete** [comment, marked as answer]
<p>Hi!</p>
<p>You can listen to the <code>check_suite</code> event and check the <code>action</code> field. More information here <a href="https://developer.github.com/v3/activity/events/types/#checksuiteevent" rel="nofollow noopener">https://developer.github.com/v3/activity/events/types/#checksuiteevent</a></p>

**Comment by dpryan79** [comment]
<p>Do you have a public repo where you have this working? A few of us have been trying this without success.</p>

**Comment by vincentpaca** [comment]
<p>Hi <a class="mention" href="/dpryan79">@dpryan79</a> , any chance you’ve found a solution for this? I’m not sure why this was marked as answered when clearly just adding the suggestion doesn’t really solve the problem.</p>

**Comment by dpryan79** [comment]
<p>Nope, I wish I had.</p>

**Comment by Cookieees** [comment]
Guys, has anyone been able to use `check_suite`? I'm already having doubts that this is what I need. I'm already automatically sending a PR comment with a workflow errors, and now I need to remove all comments with errors automatically created in the first step when all checks are passed (all workflows are successfully completed)

**Comment by kelzceana** [comment]
did anyone figure this out?

**Reply by ktarmyshov** [reply]
yeah I'm here as well :)
I have 6 checks, the longest taking is CodeQL. Checked check_run and check_suite - these are executed couple of times, when bots finish their job. But no event is triggered, when other workflows finish their jobs.
This is a big mystery to me, why this question is marked "answered", cause that suggestion does not apply.

**Comment by amatkivskiy** [comment]
Not working for me too 😞
